include/librealsense2/hpp/rs_frame.hpp

```diff
@@ -146,16 +221,27 @@ namespace rs2
             }
         }
 
+		/**
```

Comment:
Please do not add comments for "getters"/"setters" as it does not provide additional info. Alternatively put additional information,like specifying that the return parameter is in pixels, for instance.
The same applies for "getters" below